docs/src/main/ruby/generate_readme.sh

```diff
@@ -18,7 +18,7 @@ file = ARGV[0] if ARGV.length>0
 
 srcDir = File.dirname(file)
 out = "// Do not edit this file (e.g. go instead to docs/src/main/asciidoc)\n\n"
-doc = Asciidoctor.load_file file, safe: :safe, parse: false
+doc = Asciidoctor.load_file file, safe: :safe, parse: false, attributes: 'allow-uri-read'
```

Comment:
I actually don't recall why I put that in there. I think I got it from spring-boot or sagan. I'll double check to see if it does anything.